src/client/network.ts

```diff
@@ -652,7 +652,8 @@ export class RawHeaders implements api.Headers {
   }
 
   get(name: string): string | null {
-    return this.getAll(name)[0] || null;
+    const values = this.getAll(name) || [];
```

Comment:
personal opinion: this follows Headers.get from the Fetch API but seems not expected as a user. In Go e.g. I'm used that it returns [the first value](https://pkg.go.dev/net/http#Header.Get).

(feel free to resolve it)